### ISSUE TITLE
Performance measures

### ISSUE THREAD

**Issue by alasdairtran** [chengsoonong/mclass-sky#30]
There are two main measures to consider:

* Plotting the accuracy rate as a function of ra and dec. But the only class that changes over the coordinates are stars (due to the Milky Way band). Thus if we see drift over the coordinates, that means our calibration is not good, or there are some other factors that cause the classes to change with coordinates which we haven't taken into account. In any case, this plot is still a good sanity check.

* Plotting the entries in the confusion matrix as a function of ra, dec, and the features. In particular, we actually know that between quasars and stars, there is a confusion region in the feature space.

**Comment by chengsoonong** [comment]
In your confusion matrix, it is unclear what the rows and columns are referring to. I had to work it out based on class proportions. It would be better to use a pandas table, so that we have names for the indices.

**Comment by alasdairtran** [comment]
Sure of course. Will make a pandas table in the next push. The current confusion matrix follows the [sklearn convention](http://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html), where the entry in row i and column j is the number of observations known to be in group i but predicted to be in group j, and where labels are ordered alphabetically (Galaxy, Quasar, Star).

**Comment by chengsoonong** [comment]
This is solved in commit 73815259a661e99f211a7904ade0bb5959faaab9
